packages/testing/scripts/compile-proto.mjs

```diff
@@ -47,6 +47,20 @@ async function compileProtos(protoPath, jsOutputFile, dtsOutputFile, ...args) {
   ];
   await promisify(pbjs.main)(pbjsArgs);
 
+  // Workaround an issue that prevents protobufjs from loading 'long' in Yarn 3 PnP
+  // https://github.com/protobufjs/protobuf.js/issues/1745#issuecomment-1200319399
+  const pbjsOutput = readFileSync(jsOutputFile, 'utf8');
```

Comment:
You'e have to fix this in the protos build script, not everyone uses the testing package.

This specific script will go away once we take the java test server protos from core (which now embeds it).

Reply:
Ehh… not sure what you mean here… I _did_ fixed _both_ proto and testing… In proto, I injected in root.js… seems cleaner and is just as effective. Do you mean that we should not fix testing at all?

Reply:
Sorry, I missed that for some reason.